packages/twenty-front/src/modules/action-menu/types/ActionMenuType.ts

```diff
@@ -0,0 +1 @@
+export type ActionMenuType = 'recordIndex' | 'recordShow';
```

Comment:
why do we need this? I think it's a bit broken from a dependency perspective action-menu is a component that should not have to know about where it's used